### PR TITLE
Update con4gis metadata

### PR DESCRIPTION
**Pull request by coastforge-cro**

### CODE REVIEW

File: meta/con4gis/data/de.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+de:
+    title: con4gis-Data
+    description: >
+        Der Data Baustein ermöglicht es, mit wenigen Schritten ein kleines, individuelles und pflegbares Datenmodell ins Contao Back end zu bringen.
```
Comment (fritzmg) [inline meta/con4gis/data/de.yml]:
```suggestion
        Der Data Baustein ermöglicht es, mit wenigen Schritten ein kleines, individuelles und pflegbares Datenmodell ins Contao Backend zu bringen.
```

File: meta/con4gis/data/de.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        Daten können im Back end oder auch in einem mitgelieferten Frontendmodul gepflegt werden.
```
Comment (fritzmg) [inline meta/con4gis/data/de.yml]:
```suggestion
        Daten können im Backend oder auch in einem mitgelieferten Frontendmodul gepflegt werden.
```

**Comment by fritzmg** [comment]
Also you should rebase to the current master branch ;)

**Comment by coastforge-cro** [comment]
How can I rebase after I already merged the current master into my master branch? I can't figure it out right now.

**Pull request merged**